src/ui/Spinner/Spinner.tsx

```diff
@@ -15,17 +16,18 @@ const Spinner: React.FC<Props> = React.memo(
     thicknessWidth = 'w-5',
     thicknessHeight = 'h-5',
     className = '',
+    backgroundColor = 'bg-inherit',
```

Comment:
```suggestion
    backgroundColor = 'bg-[inherit]',
```

Так как у тебя - не работает в других местах нормально